.github/workflows/ci.yml

```diff
@@ -66,23 +42,85 @@ jobs:
         run: |
           curl -fsSL https://install_ao.g8way.io | bash
 
-      - name: 📥 Download deps
+      - name: 📥 Download Process deps
         working-directory: process
         run: |
           yarn
 
-      - name: 🏗 Build Module
+      - name: 📥 Download Extensions deps
+        working-directory: extensions/weavedrive
+        run: |
+          yarn
+
+      - name: 🏗 Build Process Module
         working-directory: process
         run: |
           /home/runner/.ao/bin/ao build
 
       - name: ⚡ Run Tests
-        working-directory: process
         run: |
-          yarn test
+          yarn test-all
```

Comment:
If you're going to run all tests here, it might be best to remove the `test` job in the `weavedrive.yml`